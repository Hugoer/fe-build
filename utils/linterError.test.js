const linterError = require("./linterError");
const path = require('path');
process.argv.push('--quiet');

describe('Test utils/linterError.js', () => {
    it('Check formating of a error send to console log', () => {
        const source = __filename;
        const line = 12;
        const column = 10;
        const text = 'my error'
        console.error = jest.fn();
        linterError({line, column, text}, {source});
        expect(console.error.mock.calls[0][0]).toMatchSnapshot();
    });
});
