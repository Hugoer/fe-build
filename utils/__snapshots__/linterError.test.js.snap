// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test utils/linterError.js Check formating of a error send to console log 1`] = `
"[2m[utils/linterError.js] [0m[31mâœ–  [0m[31m
    my error[0m[0m
    [2mLintError at linterError.test.js:12:10 
[0m[0m[2m[0m"
`;
